test/fixtures/IL-2022.json

```diff
@@ -0,0 +1,290 @@
+[
+  {
+    "date": "2022-01-17 00:00:00 -0600",
+    "start": "2022-01-16T16:00:00.000Z",
+    "end": "2022-01-17T16:00:00.000Z",
+    "name": "Tu Bishvat (Arbor Day)",
+    "type": "observance",
+    "rule": "15 Shvat",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-02-14 00:00:00 -0600",
+    "start": "2022-02-13T16:00:00.000Z",
+    "end": "2022-02-14T16:00:00.000Z",
+    "name": "Fast of Esther",
+    "type": "school",
+    "rule": "13 Adar",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-02-15 00:00:00 -0600",
+    "start": "2022-02-14T16:00:00.000Z",
+    "end": "2022-02-16T16:00:00.000Z",
+    "name": "Purim",
+    "type": "school",
+    "note": "optional paid leave",
+    "rule": "14 Adar P2D",
+    "_weekday": "Tue"
+  },
+  {
+    "date": "2022-04-11 00:00:00 -0600",
+    "start": "2022-04-10T15:00:00.000Z",
+    "end": "2022-04-11T15:00:00.000Z",
+    "name": "Aliyah Day",
+    "type": "observance",
+    "note": "Official holiday, business as usual",
+    "rule": "10 Nisan",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-04-16 00:00:00 -0600",
+    "start": "2022-04-15T15:00:00.000Z",
+    "end": "2022-04-16T15:00:00.000Z",
+    "name": "Passover",
+    "type": "public",
+    "rule": "15 Nisan",
+    "_weekday": "Sat"
+  },
+  {
+    "date": "2022-04-17 00:00:00 -0600",
+    "start": "2022-04-16T15:00:00.000Z",
+    "end": "2022-04-21T15:00:00.000Z",
```

Comment:
Passover 2022 ends at March 22

https://www.timeanddate.com/holidays/israel/2022

Reply:
Should be solved